react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts

```diff
@@ -1,7 +1,7 @@
-import { PickInfo } from "@deck.gl/core/lib/deck";
 import { RGBColor } from "@deck.gl/core/utils/color";
+import { LayerPickInfo } from "../../components/InfoCard";
 
-export interface BitmapPickInfo extends PickInfo<unknown> {
+export interface BitmapPickInfo extends LayerPickInfo {
```

Comment:
Nothing from the LayerPickInfo is being filled in in the bitmap layer, so probably it doesn't need to extend it.
From the current usecase, LayerPickInfo includes information needed only for the Wells layer, so you can maybe add the fields to the WellPickInfo struct directly.

Reply:
I planned to keep the common PickInfo type for all the layers required by the new info card.
But true, from the current use case only the wells layer needs this.